### ISSUE TITLE
Renaming : `app_settings.tasks.schedules[0].events` should be `app_settings.tasks.schedules[0].tasks`

### ISSUE THREAD

**Issue by estellecomment** [medic/medic-webapp#2268]
Here's `app_settings.tasks.schedules` : the term `task` is used everywhere but in that field which is called `event`.

![screen shot 2016-05-03 at 15 15 17](https://cloud.githubusercontent.com/assets/911434/14984219/e4da35d0-1141-11e6-9431-4ac8cb39f1cd.png)

Because of the design work on #1784, we'd also like to free up the term `event` to call the thing that happens that triggers a schedule. E.g `event` = new pregnancy registered, triggers a `schedule` made up of `tasks`.

**Comment by nice-snek** [comment]
hi frand @sglangevin, please [triage](https://github.com/medic/medic-docs/blob/master/md/dev/workflow.md#triaging-old-issues) before the end of this sprint.
```
--..,_                        _,.--.
       ''.'.                .''__ o  ';__.
          '.'.            .'.''  '---''  '
            '.''--....--''.'
              ''--....--''
```

**Comment by sglangevin** [comment]
We haven't yet added the schedules page back into v2.x. I'll leave this one scheduled so that it can be done with #1784.

**Comment by dianabarsan** [comment]
Hi @garethbowen. This seems to be about a configuration page which has since been abandoned/removed. Is this correct? If so, is it safe to close this issue?

**Comment by SCdF** [comment]
@dianabarsan this page still exists, but it is going to be removed in 3.0

However, the descriptions still exist in kanso.json, and while we're going to drop kanso I doubt we'd drop the configuration schema.

*However*, renaming config (which it looks like this ticket is about) seems like a large step to go down to clarify some naming. I'm not sure we want to do that.

**Comment by sglangevin** [comment]
I don't think this is worth having to update configuration across every project. That's a lot of effort so unless we are confident that this is causing confusion, I'd say leave it as-is.
